### DISCUSSION TITLE
如何不使用HC为默认控件提供的配色？

### DISCUSSION THREAD

**Discussion by LHTXQ** [HandyOrg/HandyControl#1316] [Q&A] [answered]
我使用了两个UI框架，其中一个是HC（只想使用HC一些特殊的控件，而不是修改了主题的默认控件），这两个都对WPF默认控件配色有修改，但我想让默认控件具有另一个UI框架指定的配色，而不是HC的，请问如何实现？（尝试了更改添加资源文件的先后顺序，一些顺序组合会导致无法启动，剩下的组合并不能实现我的目的）
       是否能在xaml或者代码中显式指定默认控件所使用的主题？
       可以看到下图中默认的TreeView和GroupBox控件都是使用的HC的主题，和整体不搭（窗口当前为亚克力材质），控件是纯色的。

![image](https://user-images.githubusercontent.com/38943319/205322390-83920f72-8e0d-40fe-976b-5054be311231.png)

       而HC没有定义配色的控件，显示效果就很令人满意。

![image](https://user-images.githubusercontent.com/38943319/205323808-c81138f1-3c96-4caf-930d-89be8fb83b0f.png)

**Comment by NaBian** [comment, marked as answer]
单独为 hc 控件的父级控件的 resource 添加 hc 资源，而不是在 app.xaml 中添加资源。
